packages-new/components/range/src/components/osds-range/osds-range.scss

```diff
@@ -7,10 +7,18 @@
 @import '~@ovhcloud/ods-theming/ods-theme';
 
 :host {
-  display: inline-flex;
   flex-flow: column;
-  width: var(--width);
   position: relative;
+
+  @include osds-range-on-selected-host() {
```

Comment:
this include is not necessary, it basically just select `:host`, just pu the css here directly